app/src/app/constants/colors.ts

```diff
@@ -27,7 +27,7 @@ import {
   sky,
 } from "@radix-ui/colors";
 
-export const palette = {
+const palette = {
```

Comment:
PP: If this palette is no longer being used I would remove it. We can always grab it from the git history if we want it later.